app/Image/FlysystemFile.php

```diff
@@ -0,0 +1,136 @@
+<?php
+
+namespace App\Image;
+
+use Illuminate\Contracts\Filesystem\Filesystem;
+use League\Flysystem\AdapterInterface;
+
+/**
+ * Class FlysystemFile.
+ *
+ * This class is based on legacy Flysystem v1 which ships with Laravel 8.
+ * Laravel 9 will migrate to Flysystem v2 which provides a different and
```

Comment:
I already started looking at Laravel 9, we can't do the migration yet because markdown package is not compatible yet.